lab_5_scrapper/scrapper.py

```diff
@@ -140,10 +290,74 @@ def get_search_urls(self) -> list:
         Returns:
             list: seed_urls param
         """
+        return self.config.get_seed_urls()
 
 
 # 10
 # 4, 6, 8, 10
+class CrawlerRecursive(Crawler):
+    """
+    Recursive Crawler is a child of Crawler class.
+    """
+    def __init__(self, config: Config) -> None:
+        super().__init__(config)
+
+        self.start_url = self.base_url
```

Comment:
we always start with the same link? why do not we start from the page that we visited last before the system crached/we interrupted the program with keyboard?

Reply:
If it's a child page, we may never return to the parent, which can lead to immediate crash, if those child pages do not lead to article urls.

I may think of a way to save its parent also, but for now that's my thought process

Reply:
I reworked it in the end